.circleci/config.yml

```diff
@@ -665,10 +667,14 @@ jobs:
     steps:
       - git-checkout
       - restore-workspace
+      - run:
+          name: Build storybooks
+          command: |
+            npx nx run-many -t build-storybook
       - run:
           name: Build and deploy Storybooks
           command: |
-            STORYBOOKS_USE_YARN_WORKSPACES=false npx github:mozilla-fxa/storybook-gcp-publisher
+            STORYBOOKS_USE_YARN_WORKSPACES=false LOG_LEVEL=TRACE npx github:mozilla-fxa/storybook-gcp-publisher
```

Comment:
The logs are just in the CI output fro the `Build and deploy Storybooks` storybook step.

The github:mozilla-fxa/storybook-gcp-publisher, assumes a `yarn workspace build-storybook` command produces complete output. We now need to call the nx command to ensure that a task and its dependencies are all executed or restored from cache. The switch to `nx` is where this broke. `build-storybook` now depends on build, and in the case of `fxa-auth-server`, the build produces css. 

The way `github:mozilla-fxa/storybook-gcp-publisher` is designed, has a bit of an anti pattern. It's sort of doing too much, and has a bit of feature envy making an assumption about how storybooks are built on the repository it's acting on. Also, the package name is a bit misleading since really builds and publishes. 

At any rate, the library does support the option of just publishing and not building, which I think makes sense in this case. I actually forgot that I wanted to add the `STORYBOOKS_SKIP_BUILD=true` option now that we run the build ourselves. I'll add this in report back.

Reply:
Okay, just updated this. Thanks for the questions!